pkg/sync/destination.go

```diff
@@ -76,6 +92,11 @@ func NewImageDestination(registry, repository, tag, username, password string, i
 	}, nil
 }
 
+// isPermanentServiceAccountToken returns true if user is a Google permanent service account token
+func isPermanentServiceAccountToken(registry string, username string) bool {
+	return strings.Contains(registry, ".gcr.io") && strings.Compare(username, "_json_key") == 0
```

Comment:
Could you please use a const for the `"_json_key"` here? And maybe `"_json_key"` is not intuitive enough, maybe something like `"_oauth2_"` is better.

Reply:
Done, with "_oauth2_".